apps/condo/domains/notification/constants/errors.js

```diff
@@ -19,5 +20,6 @@ module.exports = {
     EMPTY_NOTIFICATION_TITLE_BODY_ERROR,
     MESSAGE_TYPE_IN_USER_BLACK_LIST,
     MESSAGE_TYPE_IN_ORGANIZATION_BLACK_LIST,
+    MESSAGE_TYPE_IN_APP_BLACK_LIST,
```

Comment:
it's not a part of this domain logic. You don't need to change anything inside the notification domain!

Reply:
it's not a problem that we will have different blacklists that work differently? 

already existing `MESSAGE_TYPE_IN_USER_BLACK_LIST` and `MESSAGE_TYPE_IN_ORGANIZATION_BLACK_LIST` will change the status of messages in the Message table when the wokers on. `MESSAGE_TYPE_IN_APP_BLACK_LIST` will probably block adding messages to the table, if I change it as you suggest